### PR TITLE
Internal/add admin api

### PR DESCRIPTION
**Pull request by JCastrejonE**

### CODE REVIEW

File: src/Vault.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export class Vault {
         contractState: vaultState.coldState.contractState.toNumber(),
         feeStructure: vaultState.coldState.feeStructure.map((feeSplit: any) => feeSplit.toNumber()),
         minimumDebtAmount: vaultState.coldState.minimumDebtAmount.toNumber(),
+        // maximumCollateralValue: vaultState.coldState.maximumCollateralValue.toNumber(),
```
Comment (AwolDes) [inline src/Vault.ts:48]: Carry over comment from client - should add a TODO re: rc-0.0.25

File: src/__tests__/xbacked-test.ts [hunk 2]
```diff
@@ -205,3 +206,20 @@ it('Liquidator drips interest', async function () {
   });
   expect(dripInterest).toBe(true);
 });
+
+it('Admin set propoerties', async function () {
```
Comment (AwolDes) [inline src/__tests__/xbacked-test.ts:210]:
```suggestion
it('Admin set properties', async function () {
```
smol typo

File: src/__tests__/xbacked-test.ts [hunk 3]
```diff
@@ -205,3 +206,20 @@ it('Liquidator drips interest', async function () {
...
+      stabilityPoolAddress: 'CKFJQPYSGJBRDZ7YKJSOTWJOLUBM7HGIPY6MFLQTBFLHHPIOAX3VEZQP44',
+      govStakersAddress: 'CKFJQPYSGJBRDZ7YKJSOTWJOLUBM7HGIPY6MFLQTBFLHHPIOAX3VEZQP44',
+      treasuryAddress: 'CKFJQPYSGJBRDZ7YKJSOTWJOLUBM7HGIPY6MFLQTBFLHHPIOAX3VEZQP44',
+      feeStructure: [1, 5, 50, 50],
+      minimumDebtAmount: 10,
+      // maximumCollateralValue: 10000,
+    },
+    vault: new Vault({ id: VAULT_ID }),
+  });
+  expect(res).toBe(true);
```
Comment (AwolDes) [inline src/__tests__/xbacked-test.ts:224]: We should also `expect(stateAfter).toBeEqual(expectedState)` to ensure the update worked.

**Review by AwolDes — CHANGES_REQUESTED** [review]
Small update needed for the test case, otherwise this is a good addition!

**Review by Jesulonimi21 — APPROVED** [review]
Good and easy to read. Nice work

**Pull request merged**